jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1098,6 +1098,30 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _sort(*operand: tf.Tensor, dimension: int = -1,
```

Comment:
Shouldn't operand be of type `TfVal`, which includes `tf.Tensor` but also numpy arrays.